resources/views/components/inputs/password.blade.php

```diff
@@ -0,0 +1,42 @@
+<div x-data="wireui_show_password" {{ $attributes->only('wire:key') }}>
+    <x-dynamic-component
+        :component="WireUi::component('input')"
+        {{ $attributes->whereDoesntStartWith(['wire:key'])->except($except) }}
+        :borderless="$borderless"
+        :shadowless="$shadowless"
+        :label="$label"
+        :hint="$hint"
+        :corner-hint="$cornerHint"
+        :icon="$icon"
+        :prefix="$prefix"
+        :prepend="$prepend"
+        ::type="type()"
```

Comment:
Prefer to use the `x-bind:foo` when need to bind an alpine attribute into a blade component
```suggestion
        x-bind:type="type"
```

Reply:
Done!